packages/near-api-js/src/account.ts

```diff
@@ -46,25 +46,6 @@ export interface AccountAuthorizedApp {
     publicKey: string;
 }
 
-/**
- * Options used to initiate sining and sending transactions
- */
-export interface SignAndSendTransactionOptions {
-    receiverId: string;
-    actions: Action[];
-    /**
-     * Metadata to send the NEAR Wallet if using it to sign transactions.
-     * @see {@link RequestSignTransactionsOptions}
-     */
-    walletMeta?: string;
-    /**
-     * Callback url to send the NEAR Wallet if using it to sign transactions.
-     * @see {@link RequestSignTransactionsOptions}
-     */
-    walletCallbackUrl?: string;
```

Comment:
It turns out these two are used in the `Account.signAndSendTransaction` inheritance chain despite not being referenced in the base implementation directly on `Account`. We're better off keeping this interface here and narrowing the interface used by `TransactionSender` to omit these two arguments.

I'll make those changes in the base branch and rename the interfaces in `TransactionSender` to have less terrible names for what they are now.

Reply:
hmm AFAICT it's being used in wallet-account.ts. This likely warrants a new interface in that scope:

```typescript
interface WalletTransactionOptions extends Omit<SignAndSendTransactionOptions, 'signerId'> {
    /**
     * Metadata to send the NEAR Wallet if using it to sign transactions.
     * @see {@link RequestSignTransactionsOptions}
     */
    walletMeta?: string;
    /**
     * Callback url to send the NEAR Wallet if using it to sign transactions.
     * @see {@link RequestSignTransactionsOptions}
     */
    walletCallbackUrl?: string;
}
```

Reply:
I'd prefer to leave the existing interface as is and decouple these interfaces here. The fact that the interface in `account.ts` has grown to accommodate child classes indicates some drift that we don't want associated with the basic concept of transaction signing. Given that we want to ultimately break up this class hierarchy I think some small overlap is OK in the interim.